tools/app/login-redirect.hbs

```diff
@@ -1,9 +1,21 @@
-<p style="margin-bottom:10px">
-  Click the "Login with Okta" link, and use these credentials:<br />
-  User: <strong>george</strong><br />
-  Pass: <strong>Asdf1234</strong><br />
+<p>
+  Click <strong>Login with Okta</strong> to redirect to your Okta org for
+  authentication.
 </p>
-
+<table class="ui collapsing celled table compact inverted grey">
+  <thead>
+    <tr>
+      <th colspan="2">If you're using the mock-okta server:</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr><td>User</td><td><strong>george</strong></td></tr>
```

Comment:
Not sure yet - will revisit this later when we're adding more scenarios. Original reason was to disambiguate between requests (mock-server records traffic and stores it), but there's enough metadata in the headers so that's not really a problem. Mostly a philosophical question now - do we want to expose different information in the logged in state with different users to showcase different things?

Oh, now that I'm thinking about it - another reason is that when recording + at some point in the future when we have a set of tests that run against a real server - we run into rate limiting problems with one user shared between tests.